examples/05continuousrecognition.html

```diff
@@ -29,7 +27,9 @@
 		text("draw: up, down, left, right, clear", 20, 20);
 
 		myRec.onResult = parseResult; // recognition callback
-		myRec.start(); // start engine
+		
+		// start with do continuous recognition and allow partial recognition (faster, less accurate)
```

Comment:
For an example I think it might be more readable like:

```javascript
var continuous = true;
var interimResults = true;
myRec.start(continuous, interimResults)
```